### DISCUSSION TITLE
UI

### DISCUSSION THREAD

**Discussion by StefftheEmperor2** [pyload/pyload#4241] [Suggestions]
As the different UIs are a hell for development and the pyload android app isn't working anymore because of the lacking thrift-backend, i think it would be a good idea to write some sort of websocket backend/interface. Then we could rip all the frontend stuff out of pyload(-core) and one could write an fancy modern frontend in react/angular/whateveryouwant. on the other hand there was no need for some more tech-stack for the android app.
what do you think?

**Comment by GammaC0de** [comment]
I agree and I this idea of making an angular frontend for a long time ago but there is always something to do that is more important that keeps me busy..
